doc/README_macOS.md

```diff
@@ -22,8 +22,6 @@ right click the `Coq_Platform` app in `/Applications` in Finder and select `open
 - In case you want to use the installed `coqc` from the command line, please add the folder `/Applications/Coq_Platform_2022.01.0.app/Contents/Resources/bin` to your `PATH`.
 - If you want to inspect the installed content, right click the `Coq_Platform` app in `/Applications` in Finder and select `Show Package Contents`.
 
-A note to lecturers: it is easy to create a customized Windows installer from an opam switch - see [Customized Installers](#customized-installers)
```

Comment:
This change is not OK - it just deletes a useful point.

Reply:
I stripped it because this read me is about macOS and this point is included in windows read me.

Reply:
Indeed - but the point is the same for MacOS, so the right fix would be to add the corresponding section. So please leave this in and just write "MacOS" instead of "Windows", remove the link and create an issue to add the section.